src/net_processing.cpp

```diff
@@ -3804,6 +3807,25 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
                 }
             }
 
+            // Check for rebroadcasts
+            const auto current_time = GetTime<std::chrono::seconds>();
+
+            if (pto->m_next_rebroadcast < current_time) {
+                bool fFirst = (pto->m_next_rebroadcast == (std::chrono::seconds)0);
```

Comment:
nit: ` bool fFirst = (pto->m_next_rebroadcast.count() == 0);` is probably better.